apps/website/src/components/show-and-tell/ShowAndTellEntryForm.tsx

```diff
@@ -321,38 +381,35 @@ export function ShowAndTellEntryForm({
           <div className="flex items-center gap-8">
             <label
               htmlFor="giveAnHourTracked"
-              className="flex cursor-not-allowed items-center gap-4"
+              className={classes(
+                "flex items-center gap-4",
+                !canTrackGiveAnHour && "cursor-not-allowed",
+              )}
             >
               <input
                 type="checkbox"
                 id="giveAnHourTracked"
-                name="giveAnHourTracked"
-                disabled
+                disabled={!canTrackGiveAnHour}
+                checked={wantsToTrackGiveAnHour}
+                onChange={(e) => setWantsToTrackGiveAnHour(e.target.checked)}
               />
-              Yes, I want to track my hours
+              Yes, I want to track my hours:
             </label>
 
-            {/* Text input so we can show "xx hours" */}
-            {/* TODO: This will need an onChange callback to parse/format any provided value? */}
-            <TextField
-              label=""
-              name="giveAnHour"
-              // defaultValue={
-              //   ![undefined, null].includes(entry?.giveAnHour)
-              //     ? `${entry.giveAnHour} hour${entry.giveAnHour === 1 ? "" : "s"}`
-              //     : undefined
-              // }
-              placeholder="0 hours"
-              isDisabled
-              inputClassName="cursor-not-allowed"
-              pattern="^\d+ hours?$"
+            <GiveAnHourInput
+              enabled={enableTrackGiveAnHour}
+              defaultValue={
+                entry?.volunteeringMinutes ? entry.volunteeringMinutes / 60 : 1
+              }
             />
           </div>
 
-          <p className="text-sm italic opacity-75">
-            <strong>Give an Hour</strong> tracking will be available from March
-            1st.
-          </p>
+          {!canTrackGiveAnHour && (
+            <p className="text-sm italic opacity-75">
+              <strong>Give an Hour</strong> tracking will be available from
+              March 1st.
```

Comment:
Silly nitpick, should the date here come from the config? And maybe handling for the time being after the date as well with a 'tracking was available from x to y'?

Reply:
I added start and end time and kept the info visible always (before, during, and after), using the default mm/dd/yyyy date formatting. I hope thats okay?